pydantic/config.py

```diff
@@ -857,5 +857,12 @@ class Model(BaseModel):
     ```
     """
 
+    use_attribute_docstrings: bool
+    """
+    Whether attributes docstring should be used for `FieldInfo.description`.
```

Comment:
```suggestion
    Whether docstrings of attributes (bare string literals immediately following the attribute declaration)
    should be used for field descriptions.
    
    ```py
    from pydantic import BaseModel, ConfigDict, Field
    
    class Model(BaseModel):
      model_config = ConfigDict(use_attribute_docstrings=True)
    
      x: str
      """
      Example of an attribute docstring
      """
    
      y: int = Field(description='Description in Field')
      """
      Description in Field overrides attribute docstring
      """
    
    print(Model.model_fields['x'].description)
    #> Example of an attribute docstring
    print(Model.model_fields['y'].description)
    #> Description in Field
    ```

    This requires the source code of the class to be available at runtime.
```